code/modules/clothing/spacesuits/flightsuit.dm

```diff
@@ -34,7 +34,7 @@
 	var/flight = FALSE
 	var/flight_passflags = PASSTABLE
 	var/powersetting = 1
-	var/powersetting_high = 3
+	var/powersetting_high = 6
```

Comment:
Probably going to revert this once I figure out how to maths drain the cell reasonably.